sprint-mission-7/queries.sql

```diff
@@ -0,0 +1,146 @@
+/*
+  다음 경우들에 대해 총 14개의 SQL 쿼리를 작성해 주세요.
+  예시로 값이 필요한 경우 적당한 값으로 채워넣어서 작성하면 됩니다. 
+*/
+
+/*
+  1. 내 정보 업데이트 하기
+  - 닉네임을 "test"로 업데이트
+  - 현재 로그인한 유저 id가 1이라고 가정
+*/
+  UPDATE user
+  SET nickname = 'test'
+  where id = 1;
+
+/*
+  2. 내가 생성한 상품 조회
+  - 현재 로그인한 유저 id가 1이라고 가정
+  - 최신 순으로 정렬
+  - 10개씩 페이지네이션, 3번째 페이지
+*/
+  SELECT *
+  FROM product
+  WHERE user_id = 1
+  ORDER BY created_at DESC
+  LIMIT 10 OFFSET 20;
+
+/*
+  3. 내가 생성한 상품의 총 개수
+  - 현재 로그인한 유저 id가 1이라고 가정
+*/
+  SELECT COUNT(*) AS total
+  FROM product
+  WHERE user_id = 1;
+
+/*
+  4. 내가 좋아요 누른 상품 조회
+  - 현재 로그인한 유저 id가 1이라고 가정
+  - 최신 순으로 정렬
+  - 10개씩 페이지네이션, 3번째 페이지
+*/
+  SELECT P.*
+  FROM product_like AS PL
+    JOIN product AS P
+    ON PL.product_id = P.id
+  WHERE PL.user_id = 1;
```

Comment:
where 구문 마지막에 세미콜론이 들어가 있네요. 세미콜론이 있으면 쿼리가 끝났다는 의미가 되어서 뒤에 쿼리문이 동작하지 않습니다.

```suggestion
  WHERE PL.user_id = 1
```